Glyssen/Rules/Proximity.cs

```diff
@@ -39,28 +70,29 @@ public MinimumProximity CalculateMinimumProximity(ISet<string> characterIdsToCal
 			Block prevBlock = null;
 			bool breakOutOfBothLoops = false;
 
-			ISet<string> standardCharacterIdsToTreatAsOne = null;
+			//ISet<string> standardCharacterIdsToTreatAsOne = null;
 
-			ISet<string> standardCharacterIdsForBook = new HashSet<string>();
+			//ISet<string> standardCharacterIdsForBook = new HashSet<string>();
 			bool calculateAnyRelatedCharacters = characterIdsToCalculate.Any(c => relChar.HasMatchingCharacterIdsOfADifferentAge(c));
 			foreach (var book in m_booksToConsider)
 			{
 				if (breakOutOfBothLoops)
 					break;
 
-				if (!treatStandardNonScriptureCharactersAsDistinct)
-				{
-					standardCharacterIdsForBook.Clear();
-					standardCharacterIdsForBook.Add(CharacterVerseData.GetStandardCharacterId(book.BookId, CharacterVerseData.StandardCharacter.BookOrChapter));
-					standardCharacterIdsForBook.Add(CharacterVerseData.GetStandardCharacterId(book.BookId, CharacterVerseData.StandardCharacter.ExtraBiblical));
-					standardCharacterIdsForBook.Add(CharacterVerseData.GetStandardCharacterId(book.BookId, CharacterVerseData.StandardCharacter.Intro));
+				var treatAsSameCharacter = m_considerSameExtrabiblicalCharacter[book];
 
-					if (standardCharacterIdsForBook.Any(characterIdsToCalculate.Contains))
-						standardCharacterIdsToTreatAsOne = new HashSet<string>();
-				}
+				////if (!treatStandardNonScriptureCharactersAsDistinct)
```

Comment:
Unless you think this commented-out code is useful for future reference, let's delete it.